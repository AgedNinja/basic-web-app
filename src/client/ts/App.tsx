import * as React from 'react';
import '../style/app.scss';

export default class App extends React.Component<
  {},
  {}
  > {
  constructor(props: App['props']) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <div>
        <h1>{`Hello world!`}</h1>
      </div>
    );
  }
}
