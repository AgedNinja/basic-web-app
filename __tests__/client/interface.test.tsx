import * as React from 'react';
import * as renderer from 'react-test-renderer';
import App from "../../src/client/ts/App";

describe('Default', () => {
    test('displaying "fail"', () => {
        const tree = renderer
            .create(<App />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});