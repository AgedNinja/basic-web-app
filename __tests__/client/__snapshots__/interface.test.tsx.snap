// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default displaying "fail" 1`] = `
<div>
  <h1>
    Hello world!
  </h1>
</div>
`;
